x-pack/plugins/security_solution/public/common/components/query_bar/index.tsx

```diff
@@ -167,7 +167,6 @@ export const QueryBar = memo<QueryBarComponentProps>(
         savedQuery={savedQuery}
         displayStyle={displayStyle}
         isDisabled={isDisabled}
```

Comment:
Small thing I noticed that has probably always been there. When you update the query in either the main security application or in discover the `refresh` button changes to `update`. Would it be a huge lift to implement that here as well? If so, we can pull it out to a separate issue

Reply:
@michaelolo24 , I can see it  happening today.


https://github.com/elastic/kibana/assets/7485038/dd97e743-9e12-4b87-a623-8efcd2815327

Reply:
Oh yea, I agree it's not a bug you introduced. Was just wondering if it could be done as part of this refactor to align with the other experiences, but that can be done separately.